index.html

```diff
@@ -0,0 +1,64 @@
+<!-- landing page for personal portfolio -->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title> Tamiko Terada </title>
+    <link rel="stylesheet" type="text/css" href="stylesheets/main.css">
+    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css">
+  </head>
+
+  <body>
+    <a href="html/menu.html" target="_parent"><button autofocus id="test"> </button></a>
+    <!-- header | name, image, banner, button-link -->
+    <header>
+      <div class="left-bar">
+        <p class="top-bar">Tamiko Terada</p>
+      </div>
+      <nav><a href="menu.html">menu</a></nav>
+    </header>
+
+    <div id="header-image"></div>
+    <section>
+      <div class="box-green">
+        <p class="centered"><a class="button" href="https://www.linkedin.com/in/tamikoterada/" target="_blank">TAMIKO has skills.</a>
```

Comment:
Like with your name in the header, is this really a paragraph? Or more of a header? `<h1></h1>`